gameyamlspiderandgenerator/util/spider.py

```diff
@@ -16,31 +17,25 @@
 def get_json(url: AnyStr) -> SupportsInt | Dict:
     logger.info(setting)
     try:
-        response = requests.get(url, proxies=setting['proxy'])
-        if response.status_code == 200:
-            return response.json()
-        else:
-            return response.status_code
+        response = requests.get(url, proxies=setting["proxy"])
+        return response.json() if response.status_code == 200 else response.status_code
     except Exception as e:
         logger.error(e)
         return -3
 
 
 def get_text(url: AnyStr) -> SupportsInt | AnyStr:
     try:
-        response = requests.get(url, proxies=setting['proxy'])
-        if response.status_code == 200:
-            return response.text
-        else:
-            return response.status_code
+        response = requests.get(url, proxies=setting["proxy"])
+        return response.text if response.status_code == 200 else response.status_code
     except Exception as e:
         logger.error(e)
         return -3
 
 
 def get_status(url: AnyStr) -> SupportsInt:
     try:
-        return requests.get(url, proxies=setting['proxy']).status_code
+        return requests.get(url, proxies=setting["proxy"]).status_code
     except Exception as e:
         logger.error(e)
         return -3
```

Comment:
```suggestion
        raise SomeException(-3)
```

1. Raise exception instead of returning integer